src/tir/transforms/lower_sparse_tir.cc

```diff
@@ -224,8 +285,10 @@ class IndexTransformer : public StmtExprMutator {
   PrimExpr AccumulateLowerIndex(PrimExpr prev_lowered_index, const SparseBuffer& sp_buffer, int dim,
                                 PrimExpr index) {
     const Axis& axis = sp_buffer->axes[dim];
-    if (axis->IsInstance<DenseFixedAxisNode>() || axis->IsInstance<SparseFixedAxisNode>()) {
```

Comment:
ditto?

Reply:
Err, no. This is different from the comment above. The change is a bug-fix. Before this PR I mistakenly use `length` for sparse-fixed axes, but we need rather use `num_cols`.